Sources/Sentry/Public/SentrySpanStatus.h

```diff
@@ -1,3 +1,4 @@
+#import <Foundation/Foundation.h>
```

Comment:
m: why do we need this here?

Reply:
Because, weirdly, typedef depends on foundation.

So, if we include the reference to SentrySpanStatus.h in another header that does not contains the reference to Foundation the project will not compile.

Happened with SentrySampleDecision.h

Reply:
thanks